### ISSUE TITLE
When click outside of drawer not closing

### ISSUE THREAD

**Issue by juniorschen** [NativeScript/NativeScript#8779]
**Environment**
![image](https://user-images.githubusercontent.com/46031261/90694199-a1512e80-e24e-11ea-9e51-ce8c3f1b34f9.png)

**Describe the bug**
When i set the drawerContent.effectiveWidth and drawerContent.width of radsidedrawer this works fine visually, but when i click outside not closing modal.

**To Reproduce**
Set width and effectiveWidth  of drawer content;
![image](https://user-images.githubusercontent.com/46031261/90694397-effec880-e24e-11ea-8379-c2cae83da98f.png)

**Expected behavior**
When click outside close modal.

**Additional context**
The red region not close modal, this is because width has been set
![image](https://user-images.githubusercontent.com/46031261/90694479-102e8780-e24f-11ea-8da3-e5b37f23f8d6.png)

**Comment by Ross-Olivares** [comment]
Hi @juniorschen 
Thank you for reporting this issue.
We are going to try replicate this case on our end. Let me know if you can send us a sample project or playground link that can reproduce the issue.

**Comment by juniorschen** [comment]
@Ross-Olivares 
Of course, sorry I didn't send it before, I did it in angular because that's where I have the domain, here it is:
https://github.com/juniorschen/Radlist-Outside

every green area closes the modal, but the red area is not because of the width
![Uploading image.png…]()

**Comment by juniorschen** [comment]
@Ross-Olivares 
Do you think there would be any quick fix? I tried to tap the area but without success :/
